consensus/polybft/polybft_config.go

```diff
@@ -44,6 +44,12 @@ type PolyBFTConfig struct {
 	// NativeTokenConfig defines name, symbol and decimal count of the native token
 	NativeTokenConfig *TokenConfig `json:"nativeTokenConfig"`
 
+	// BridgeAllowListActive indicates whether bridge allow list is active
+	BridgeAllowListAdmin types.Address `json:"bridgeAllowListAdmin"`
```

Comment:
Should we added these to the `BridgeConfig` struct?

Reply:
I was thinking on this, yes it looks cleaner

Reply:
I propose to do it in a follow-up PR

Reply:
Do we even need this data redundancy? We already should be able to have bridge access lists (together with other access lists) persisted in `*chain.Chain.Params`. From there we could extract all the information needed (owner of predicate, is given access list enabled or not etc.).
I'd like to hear your opinions as well.

Reply:
I find cleaner to separate the logic and send the appropriate value here to the GenesisPostHook but it's true that it's redundant.

Reply:
> I propose to do it in a follow-up PR

I am thinking out loud, but if we are to move this later to the Bridge config, doesn't that break the old `genesis` file? If those values were on one place in one release, and then in another place in the second release, then we break the genesis file? Do we even care at this point? Probably not.

Reply:
Good question, but I think the important part that is the Allocation are not going to move, so extra values in genesis won't be a problem and won't break genesis, I think we are good in that regard.